[Chore] Clean up Javadocs

```
/Users/widdisd/git/flow-framework/src/main/java/org/opensearch/flowframework/common/CommonValue.java:272: warning: no comment
    public static final String WORKFLOW_RESOURCE_TYPE = "workflow";
                               ^
/Users/widdisd/git/flow-framework/src/main/java/org/opensearch/flowframework/common/CommonValue.java:271: warning: no comment
    public static final String WORKFLOW_STATE_RESOURCE_TYPE = "workflow_state";
                               ^
/Users/widdisd/git/flow-framework/src/main/java/org/opensearch/flowframework/model/Config.java:109: warning: no main description
     * @return the masterKey
       ^
/Users/widdisd/git/flow-framework/src/main/java/org/opensearch/flowframework/model/Config.java:116: warning: no main description
     * @return the createTime
       ^
/Users/widdisd/git/flow-framework/src/main/java/org/opensearch/flowframework/model/Config.java:123: warning: no main description
     * @return the tenantId
       ^
/Users/widdisd/git/flow-framework/src/main/java/org/opensearch/flowframework/FlowFrameworkResourceSharingExtension.java:25: warning: use of default constructor, which does not provide a comment
public class FlowFrameworkResourceSharingExtension implements ResourceSharingExtension {
       ^
/Users/widdisd/git/flow-framework/src/main/java/org/opensearch/flowframework/util/PluginClient.java:40: warning: no comment
    public void setSubject(Subject subject) {
                ^
/Users/widdisd/git/flow-framework/src/main/java/org/opensearch/flowframework/transport/handler/SearchHandler.java:71: warning: no description for @param
     * @param resourceType
       ^
/Users/widdisd/git/flow-framework/src/main/java/org/opensearch/flowframework/workflow/ToolStep.java:41: warning: use of default constructor, which does not provide a comment
public class ToolStep implements WorkflowStep {
       ^
9 warnings
```

--- a/src/main/java/org/opensearch/flowframework/model/Config.java
+++ b/src/main/java/org/opensearch/flowframework/model/Config.java
@@ -106,6 +106,7 @@ public class Config implements ToXContentObject {
     }
 
     /**
+     * Gets the master key used for encryption.
      * @return the masterKey
      */
     public String masterKey() {
@@ -113,6 +114,7 @@ public class Config implements ToXContentObject {
     }
 
     /**
+     * Gets the creation time of the config.
      * @return the createTime
      */
     public Instant createTime() {
@@ -120,6 +122,7 @@ public class Config implements ToXContentObject {
     }
 
     /**
+     * Gets the tenant ID associated with the config.
      * @return the tenantId
      */
     public Object tenantId() {

